core/CronArchive/SharedSiteIds.php

```diff
@@ -49,11 +50,24 @@ public function __construct($websiteIds, $optionName = self::OPTION_DEFAULT)
                 return $existingWebsiteIds;
             }
 
+            $self->setQueueWasReset();
             $self->setSiteIdsToArchive($websiteIds);
 
             return $websiteIds;
         });
-        $this->numWebsitesLeftToProcess = $this->getNumSites();
+
+        $this->initialResetQueueTime = $this->getResetQueueTime();
+    }
+
+    public function setQueueWasReset()
+    {
+        Option::set($this->optionName . self::KEY_TIMESTAMP, floor(microtime(true) * 1000));
```

Comment:
Not an issue, but curious why microtime is used instead of just time()? Something to do with server configurations?

Reply:
I see 👍